avaje-jex/src/main/java/io/avaje/jex/http/BadRequestException.java

```diff
@@ -1,6 +1,6 @@
 package io.avaje.jex.http;
 
-/** Thrown when unable to find a route/resource */
+/** Thrown when request is invalid */
 public class BadRequestException extends HttpResponseException {
 
   public BadRequestException(String message) {
```

Comment:
Missing a constructor that takes jsonResponse?

I think we are also looking for a test?